edwin/client/static/js/components/TimelineApp.js

```diff
@@ -78,24 +69,31 @@ export default class TimelineApp extends React.Component {
  */
 class BugRow extends React.Component {
   render() {
+    let bug = this.props.bug;
+    let prs = this.props.prs;
+    let bugUrl = `https://bugzilla.mozilla.org/show_bug.cgi?id=${bug.get('id')}`;
+
     return (
       <tr>
         <td className="BugTable__data--number">
-          <a href={`https://bugzilla.mozilla.org/show_bug.cgi?id=${this.props.id}`}>
-            {this.props.id}
+          <a href={bugUrl}>
+            {bug.get('id')}
           </a>
         </td>
         <td className="BugTable__data">
-          {this.props.summary}
+          {bug.get('summary')}
         </td>
         <td className="BugTable__data">
-          <AssignedTo {...this.props.assigned_to_detail}/>
+          <AssignedTo user={bug.get('assigned_to_detail')}/>
```

Comment:
We almost assuredly want to edit this somehow because the bugzilla username data is silly looking in a lot of cases.

We can push that off for now, too.